live-build/misc/live-build-hooks/82-etc-apt-sources-list.binary

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -eux
+#
+# Copyright 2018 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# We specifically do not want the appliance to attempt to upgrade, nor
+# install, any packages by contacting servers listed in the default list
+# of APT repositories, found in /etc/apt/sources.list. Instead, the
+# appliance will handle upgrade differently, using a custom upgrade
+# mechanism that is capable of upgrading itself without network
+# connectivity to these external APT repositories. Thus to avoid any
+# source of confusion or mistakes, we actually remove the default
+# list(s) here.
+#
+chroot binary rm -f /etc/apt/sources.list
```

Comment:
We should skip this step for the internal-dev variant. We should however still remove localhost.list.

Reply:
We should probably do this via ansible, then, if we want it to be configured differently based on the variant. I'm also not completely sure we should keep it enabled for the developer variant, but maybe.....

Reply:
How would we do it via ansible since we need to do it after generating the repo?